source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
 Create a Clear Linux WordPress\* server
 #######################################
 
-Prerequisites
-=============
+This tutorial guides you through installing WordPress components on your |CL| LAMP server. During :ref:`Creating a Clear Linux based web server <web-server-install>`, you created the initial WordPress MySQL database `WordPress` using phpMyAdmin\* and MariaDB\*. Now that the LAMP server is up and running, you can add the WordPress components needed to host a WordPress website on your host system.
 
-This tutorial assumes you have already
-:ref:`installed Clear Linux on a bare metal system <bare-metal-install>`
-and that you have :ref:`set up a LAMP web server <web-server-install>`.
+This tutorial assumes you have successfully installed :ref:`Clear Linux on bare metal<bare-metal-install>` and that you have :ref:`set up a LAMP web server <web-server-install>`.
 
-Additionally, all the steps on this tutorial were tested using a NUC6i5SYH
```

Comment:
Add Lines 13-14 back into the tutorial. No reason to remove them.

Reply:
Added